packages/lib/passport-interface/src/genericIssuanceTypes.ts

```diff
@@ -87,6 +87,17 @@ const ManualTicketSchema = z.object({
   attendeeName: z.string().min(1)
 });
 
+const ManualTicketListSchema = z
+  .array(ManualTicketSchema)
+  .optional()
+  .default([])
```

Comment:
let's remove the `default([])` for now because it types all `manualTickets` as `ManualTicketSchema[]` rather than `ManualTicketSchema[] | undefined`, which led to the `.map()` not being typechecked, causing the pipelines 500 error